app/frontend/src/components/apiAccess/ClientStatus.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
+    {{ env }}:
+    <span :class="display.class">{{display.text}}</span>
```

Comment:
Space between the curly braces and content

Reply:
Fixed